client/components/course.jsx

```diff
@@ -14,10 +14,15 @@ export default class Course extends React.Component {
   render() {
     const courseName = this.props.name;
     const courseId = this.props.courseId;
+    const backPage = this.props.currentPage;
     return (
       <div className="w-100 d-flex justify-content-center">
         <button
-          onClick={() => this.setViewAndCourseId('myLessons', courseId)}
+          onClick={() => {
+            this.setViewAndCourseId('myLessons', courseId);
+            this.props.setBackPage(backPage);
+          }
+          }
```

Comment:
Have the onClick only call one function and call setBackPage() in the setViewAndCourseId() method. Then just rename that method with a clear/succinct name.

Reply:
Took everything in the callback code block and put it into new method 'setNextPage'